src/xdocs/config_annotation.xml

```diff
@@ -142,49 +142,44 @@ public String getNameIfPresent() { ... }
 
       <subsection name="Examples" id="AnnotationLocation_Examples">
         <p>
-          Example to allow multiple annotations on the same line
-        </p>
-        <source>
-@SuppressWarnings("deprecation") @Mock DataLoader loader; // no violations
-        </source>
-        <p>
-          Use the following configuration:
+            Default configuration - Use the following configuration to allow one single
```

Comment:
You say "Default config" but you show a config with things set to specific values and not default. 

See: https://checkstyle.org/config_naming.html#AbbreviationAsWordInName_Examples

This is a default config: `<module name="AbbreviationAsWordInName"/>`.